src/python/pants/bin/pants_exe.py

```diff
@@ -28,11 +28,12 @@ def test_env():
 def main():
   start_time = time.time()
 
+  # NB: This is the root exiter, so we don't need a contextmanager for this.
   exiter = Exiter()
   ExceptionSink.reset_exiter(exiter)
```

Comment:
As far as I'm aware, the only reason we have this `Exiter` in this file is because we want to maintain this `except KeyboardInterrupt as e` messaging behavior -- there is already an `Exiter` registered at import time at the bottom of `exception_sink.py`, which would be active here. I think we may be able to remove this exiter entirely.

The reason this wasn't done before was because I wasn't sure whether there was a specific reason we'd made this exiter in `pants_exe.py` in the first place. It doesn't have to be done in this particular change, but if we're going for exiter hygiene, I might suggest attempting to remove the `Exiter` from this file and change whatever exception string you need to to make the same tests pass.